plugins/font-awesome.js

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue';
+
+export const extend = function (config) {
+  css: [
```

Comment:
- path to node_modules should be relative to projects rootDir
- extend is just a function which mutates nuxt config.we should push this line to css.
Something like this:
```js
var fa = 'node_modules/font-awesome/css/font-awesome.css';
if (config.rootDir) {
  fa = path.resolve(config.rootDir, fa);
}
config.css.push(fa)
```